app/pages/api/[submit].js

```diff
@@ -0,0 +1,19 @@
+import { withSession } from 'next-session';
```

Comment:
This file works similarly to [page], but as an API route for post requests. This way you can post to the api to save data into the session